commands/images/reddit.js

```diff
@@ -0,0 +1,80 @@
+const { MessageEmbed, MessageActionRow, MessageButton, Guild, GuildChannel, Channel, Message } = require('discord.js');
+const snoowrap = require('snoowrap');
```

Comment:
You probably don't need to use a library. Just fetch [https://www.reddit.com/r/piracy.json](https://www.reddit.com/r/piracy.json) and parse the JSON.
On another note: You should always add dependencies to the `package.json` file!